src/components/Subscription/PublicCalendarSubscriptionPicker.vue

```diff
@@ -21,22 +21,29 @@
 
 <template>
 	<NcModal @close="$emit('close', $event)">
-		<div class="holiday-subscription-picker">
-			<h2>{{ t('calendar', 'Public holiday calendars') }}</h2>
-			<p class="holiday-subscription-picker__attribution">
-				{{ t('calendar', 'Public holiday calendars are provided by Thunderbird. Calendar data will be downloaded from {website}', { website: 'thunderbird.net' }) }}
+		<div class="public-calendar-subscription-picker">
+			<h2 v-if="showHolidays">
+				{{ t('calendar', 'Public holiday calendars') }}
+			</h2>
+			<h2 v-else>
+				{{ t('calendar', 'Public calendars') }}
+			</h2>
+			<p v-if="showHolidays" class="holiday-subscription-picker__attribution">
```

Comment:
I'd appreciate if the "custom" public calendar section also had a paragraph explaining the below calendars have been suggested by the sever administrator.